www/class/config-generate-remote/Abstracts/AbstractObject.php

```diff
@@ -162,9 +162,9 @@ protected function writeObject(array $object): void
         $append = '';
         for ($i = 0; $i < count($this->attributesWrite); $i++) {
             if (isset($object[$this->attributesWrite[$i]]) && strlen($object[$this->attributesWrite[$i]])) {
-                $line .= $append . $object[$this->attributesWrite[$i]];
+                $line .= $append . '"' . str_replace('"', '""', $object[$this->attributesWrite[$i]]) . '"';
```

Comment:
"attribute" will result to """attribute""" ?

Reply:
Actually '"' are stored like this '&quot;'.
So it will result to "&quot;attribute&quot;".

Reply:
Well, actually, not all entries are using `&quot;`.
For the other it will result to `"""attribute"""`.
And its working fine.